upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -223,6 +223,7 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap, secretStore fi.SecretS
 		return strings.Join(labels, ",")
 	}
 
+	dest["IsIPv6Only"] = tf.IsIPv6Only
```

Comment:
Same as for Calico, would you be ok with something like this? It would make the template more clear.
At this stage, it is irrelevant if cluster is IPv4 or IPv6, but only if they should be enabled or not.
Later, if we decide to add DualStack, we can keep the template and just modify the logic in the function(s).
```suggestion
	dest["EnableIPv4"] = !tf.IsIPv6Only
	dest["EnableIPv6"] = tf.IsIPv6Only
```

Reply:
Dual stack is out of scope. If and when it is implemented it will require extensive changes, not just to this.

Reply:
I never said it would not require changes or that we will ever do it, just that using `EnableIPv4` and `EnableIPv6` feels more natural in templates.

Reply:
I disagree. Which would be more natural for selecting the health probe host?

There are only two states we support; that should be evident when reading the templates.

Reply:
I am not quite sure the health probe host needs changing.

Reply:
Cilium requires the health check to explicitly use localhost. If this isn't set, the health check will use the node IP, which will fail.

Reply:
When I put Cilium into IPv6-only mode, it stopped listening to `127.0.0.1`.

Reply:
Fine by me.